style.css

```diff
@@ -127,7 +145,7 @@ ul.footer-list a {
   color: #777;
 }
 
-ul.footer-list .sr-only {
+/* ul.footer-list .sr-only {
```

Comment:
Sames as before. This declaration can be removed

Reply:
done